frontend/src/utils/filterHelpers.js

```diff
@@ -314,6 +316,10 @@ export function isFilterValid(filters) {
 export function parseFiltersToPatch(params) {
   return params.reduce((acc, param) => {
     // filters with only defaltValue shouldn't be sent to server
```

Comment:
imho this comment shall be removed together with yours.
IMHO it just introducing FUD without making things clearer for the reader.